docs/how-to/bgp.rst

```diff
@@ -0,0 +1,150 @@
+the MicroOVN, for more information about what's happening under the hood, see:
+:doc:`Explanation: OVN integration with BGP </explanation/bgp-redirect>`.
+
+Enable BGP integration
+----------------------
+
+In this example, we have a host connected to two external networks via
+interfaces ``eth1`` and ``eth2``.
+
+All we need to proceed with the configuration is an unused `VRF`_ table number
+and AS number. The AS number is an optional argument. If it's omitted, MicroOVN
+won't configure a BGP daemon. See the section below on
+:ref:`Manual BGP daemon configuration <manual_bgp>`.
+
+For this example, we'll pick VRF ``10`` and AS number from the private range,
+``4210000000``.
+
+.. important::
+
+   Never use interface that provides actual host connectivity for the purpose
+   of OVN BGP integration. These interfaces are meant for the OVN's traffic,
+   they will be assigned to a OVS bridge and you will lose your connection to the host.
+
+To enable BGP integration run:
+
+.. code-block:: none
+
+   microovn enable bgp --config ext_connection=eth1,eth2 --config vrf=10 --config asn=4210000000
+
+You will receive positive confirmation message in the CLI and the setup is
+done.
+
+Inspect the changes
+~~~~~~~~~~~~~~~~~~~
+
+If you are interested in changes that the MicroOVN made to the system, We can
+inspect the new interfaces that were created.
+
+.. code-block:: none
+
+   ip link
+
+The output will show that we have a new VRF device and two veth pairs for
+the BGP control-plane traffic (one pair for each external interface):
+
+.. code-block:: none
+
+   <snipped preceding output>
+   13: ovnvrf10: <NOARP,MASTER,UP,LOWER_UP> mtu 65575 qdisc noqueue state UP mode DEFAULT group default qlen 1000
+       link/ether 2e:25:b7:f0:f7:21 brd ff:ff:ff:ff:ff:ff
+   14: veth1-brg@veth1-bgp: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master ovs-system state UP mode DEFAULT group default qlen 1000
+       link/ether 46:d7:9e:a5:81:fa brd ff:ff:ff:ff:ff:ff
+   15: veth1-bgp@veth1-brg: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master ovnvrf10 state UP mode DEFAULT group default qlen 1000
+       link/ether 02:0e:da:a6:c4:28 brd ff:ff:ff:ff:ff:ff
+   16: veth2-brg@veth2-bgp: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master ovs-system state UP mode DEFAULT group default qlen 1000
+       link/ether ba:1e:35:32:8b:36 brd ff:ff:ff:ff:ff:ff
+   17: veth2-bgp@veth2-brg: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue master ovnvrf10 state UP mode DEFAULT group default qlen 1000
+       link/ether 02:66:5b:b1:78:6e brd ff:ff:ff:ff:ff:ff
+   <snipped remaining output>
+
+And since we requested auto-configuration of BGP daemon, we can check the
+BIRD configuration found at ``/var/snap/microovn/common/data/bird/bird.conf``.
+There should be two "bgp" instances instances.
+
+.. code-block:: none
+
+   <snipped preceding output>
+   protocol bgp microovn_eth2 {
+        router id 14.218.166.196;
```

Comment:
Let's use a number meant for use in documentation ref [RFC 5737](https://datatracker.ietf.org/doc/html/rfc5737)